src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -107,6 +112,23 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
         return response;
     }
 
+    /**
+     * PUT text-submissions/:submissionId/example-assessment : save manual example text assessment
+     *
+     * @param submissionId id of the submission
+     * @param feedbacks list of feedbacks
+     * @return result after saving example text assessment
+     */
+    @ResponseStatus(HttpStatus.OK)
+    @ApiResponses({ @ApiResponse(code = 403, message = ErrorConstants.REQ_403_REASON), @ApiResponse(code = 404, message = ErrorConstants.REQ_404_REASON) })
+    @PutMapping("/text-submissions/{submissionId}/example-assessment")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<Result> saveTextExampleAssessment(@PathVariable long submissionId, @RequestBody List<Feedback> feedbacks) {
+        log.debug("REST request to save text example assessment : {}", submissionId);
+        // TODO: Figure out if we need to use saveTextBlocks
+        return super.saveExampleAssessment(submissionId, feedbacks);
```

Comment:
@jpbernius I am not quite sure if we need to use `saveTextBlocks` here. For the example assessment, we are using the old version of assessing text exercises

Reply:
We decided that it is not needed as the client would send an empty array to the server anyways.